### DISCUSSION TITLE
Ability to set http_proxy for specific modules

### DISCUSSION THREAD

**Discussion by Sh4d0wHunt3rX** [blacklanternsecurity/bbot#1224] [General]
Maybe in future, it isn't a bad idea to have ability to set proxy for only some selected modules. For example, passive modules go directly and aggressive modules go through proxy. This will make the scan faster in compare to when we set proxy for the whole scan and limit the threads (if our proxy has thread limitation)

**Comment by TheTechromancer** [comment]
If this works out: https://github.com/blacklanternsecurity/bbot/issues/1223, we'll probably end up having two web helpers, one for scanning/fuzzing and one for passive stuff. That will make it easier to set the http proxy on just one.
